openedx/features/course_card/tests/helpers.py

```diff
@@ -1,5 +1,5 @@
 """
-Unit test helper file
+Helper file which contains the helper methods used in different Course card Unit Tests
```

Comment:
Please update docstring, this is not a file it is module

Reply:
Done